Some customization of the option which are straying from the default
only on Debian systems

--- a/init.h
+++ b/init.h
@@ -312,6 +312,9 @@
   ** .pp
   ** When this variable is \fIset\fP, mutt will include Delivered-To headers when
   ** bouncing messages.  Postfix users may wish to \fIunset\fP this variable.
+  ** .pp
+  ** \fBNote:\fP On Debian systems, this option is unset by default in
+  ** /etc/Muttrc.
   */
   { "braille_friendly", DT_BOOL, R_NONE, OPTBRAILLEFRIENDLY, 0 },
   /*
@@ -1033,6 +1036,9 @@
   ** one is not used.
   ** .pp
   ** Also see $$use_domain and $$hidden_host.
+  ** .pp
+  ** \fBNote:\fP On Debian systems, the default for this variable is obtained
+  ** from /etc/mailname when Mutt starts.
   */
   { "ignore_linear_white_space",    DT_BOOL, R_NONE, OPTIGNORELWS, 0 },
   /*
@@ -1566,6 +1572,9 @@
   ** system.  It is used with various sets of parameters to gather the
   ** list of known remailers, and to finally send a message through the
   ** mixmaster chain.
+  ** .pp
+  ** \fBNote:\fP On Debian systems, this option is set by default to
+  ** ``mixmaster-filter'' in /etc/Muttrc.
   */
 #endif
   { "move",		DT_QUAD, R_NONE, OPT_MOVE, M_NO },
@@ -2585,6 +2594,10 @@
   ** This variable contains the name of either a directory, or a file which
   ** contains trusted certificates for use with OpenSSL.
   ** (S/MIME only)
+  ** .pp
+  ** \fBNote:\fP On Debian systems, this defaults to the first existing file in
+  ** the following list: ~/.smime/ca-certificates.crt ~/.smime/ca-bundle.crt
+  ** /etc/ssl/certs/ca-certificates.crt.
   */
   { "smime_certificates",	DT_PATH, R_NONE, UL &SmimeCertificates, 0 },
   /*
@@ -2791,6 +2804,9 @@
   ** .ts
   ** set smtp_authenticators="digest-md5:cram-md5"
   ** .te
+  ** .pp
+  ** \fBNote:\fP On Debian systems, this variable defaults to the example
+  ** mentioned. This file is managed by the ``ca-certificates'' package.
   */
 # endif /* USE_SASL */
   { "smtp_pass", 	DT_STR,  R_NONE, UL &SmtpPass, UL 0 },
@@ -2838,6 +2854,9 @@
   ** .pp
   ** You may optionally use the ``reverse-'' prefix to specify reverse sorting
   ** order (example: ``\fCset sort=reverse-date-sent\fP'').
+  ** .pp
+  ** \fBNote:\fP On Debian systems, this option is set by default to
+  ** ``threads'' in /etc/Muttrc.
   */
   { "sort_alias",	DT_SORT|DT_SORT_ALIAS,	R_NONE,	UL &SortAlias, SORT_ALIAS },
   /*
@@ -3375,6 +3394,9 @@
   ** is set to deliver directly via SMTP (see $$smtp_url), this
   ** option does nothing: mutt will never write out the ``Bcc:'' header
   ** in this case.
+  ** .pp
+  ** \fBNote:\fP On Debian systems, exim4 and postfix strip BCC headers by
+  ** default. The above warning applies to exim3 users, see /etc/Muttrc.
   */
   { "write_inc",	DT_NUM,	 R_NONE, UL &WriteInc, 10 },
   /*
